awsmfa/__init__.py

```diff
@@ -273,10 +277,14 @@ def validate(args, config):
 
 
 def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
-    console_input = prompter()
-    mfa_token = console_input('Enter AWS MFA code for device [%s] '
-                              '(renewing for %s seconds):' %
-                              (args.device, args.duration))
+    if args.token:
+        logger.debug("Received token as argument")
+        mfa_token = '%s' % (args.token)
+    else:
+        console_input = prompter()
+        mfa_token = console_input('Enter AWS MFA code for device [%s] '
+                                '(renewing for %s seconds):' %
```

Comment:
286 and 287 are not properly indented

Reply:
Good catch, adjusted based on PEP8 indentation style.